### ISSUE TITLE
[Screen reader - Logic Apps - Overview]: "Azure backup custom connector" is incorrectly defined as h1 heading.

### ISSUE THREAD

**Issue by takyyon** [Azure/LogicAppsUX#6112]
### Describe the issue 

User Experience:

 Users who rely on screen readers for navigation will be impacted if headings are incorrectly defined on the screen as users will get incorrect heading information and they will get disoriented.

Note: User credentials should NOT be included in the bug.

Repro Steps:

- Open [https://ms.portal.azure.com/](<https://ms.portal.azure.com/>) and login via valid credentials. 
- Navigate to the Left Nav bar and select 'More Services', search 'Logic App Custom Connector' and select any already created ''Logic App Custom Connector' 
- Select Overview in left navigation pane. Navigate all the controls on the blade 
- Verify "Azure backup custom connector" is incorrectly defined as h1 heading. 

Actual Result:

 "Azure backup custom connector" is incorrectly defined as h1 heading present on the "Overview" screen.

Expected Result:

 "Azure backup custom connector" should be programmatically defined as h2 heading present on the "Overview" screen.

### Are you using new designer or old designer 

New Designer (With action panels)

### Screenshots or Videos 

*No response*

### Browser 

Chrome, Edge

### Additional context 

*No response*
